sympy/polys/tests/test_polytools.py

```diff
@@ -1700,6 +1700,14 @@ def test_div():
     q = f.exquo(g)
     assert q.get_domain().is_ZZ
 
+    f, g = Poly(x+y, x), Poly(2*x+y, x)
+    q, r = f.div(g)
+    assert q.get_domain().is_Frac and r.get_domain().is_Frac
+    r = f.rem(g)
```

Comment:
Maybe this rem test and the quo test below are not necessary since they are the same as the results `q, r` from div above.